packages/flutter_tools/bin/tool_backend.dart

```diff
@@ -34,21 +37,29 @@ or
     exit(1);
   }
 
-  String cacheDirectory;
-  switch (targetPlatform) {
-    case 'linux-x64':
-      cacheDirectory = 'linux/flutter/ephemeral';
-      break;
-    case 'windows-x64':
-      cacheDirectory = 'windows/flutter/ephemeral';
-      break;
-    default:
-      stderr.write('Unsupported target platform $targetPlatform');
+  if (targetPlatform == 'linux-x64') {
+    final ProcessResult unpackResult = await Process.run(
+        flutterExecutable,
+        <String>[
+          '--suppress-analytics',
+          '--verbose',
+          if (flutterEngine != null) '--local-engine-src-path=$flutterEngine',
+          if (localEngine != null) '--local-engine=$localEngine',
+          'assemble',
+          '-dTargetPlatform=$targetPlatform',
+          '-dBuildMode=$buildMode',
```

Comment:
This will fail with non-debug builds for now since there are no profile or release targets.

Reply:
ack

Reply:
Actually, I updated to be hardcoded to debug so it doesn't crash on --release